### PR TITLE
feat(message subscriptions): update message subscriptions

### PR DESCRIPTION
**Pull request by artdnlv**
In the scope of opt-ins feature development:

* added missing types on Message Subscription interface
* added `duplicate` method for a message subscription entity
* implemented `fetch` interface for `messageSubscriptions`  on `universe` to work with Data Table on agent ui

https://app.clickup.com/t/24327122/TECH-552

### CODE REVIEW

**Comment by eljefedelrodeodeljefe** [comment]
Task linked: [TECH-552 agent ui: adopt the redesign of opt-in flow](https://app.clickup.com/t/24327122/TECH-552)

File: src/entities/message-template/message-template.ts [hunk 1]
```diff
@@ -43,6 +43,45 @@ export interface MessageTemplateRawPayloadCarousel {
   items?: MessageTemplateRawPayloadCarouselItem[]
 }
 
+export interface MessageTemplateRawPayloadQuickReply {
+  type?: 'text' | string | null
```
Comment (datrinh) [inline src/entities/message-template/message-template.ts:47]: What does "text" do when it can be a string anyhow? :)

**Review by datrinh — APPROVED** [review]
LGTM 👍

**Review by haayhappen — APPROVED** [review]
LGTM

File: src/universe/index.ts [hunk 2]
```diff
@@ -299,6 +299,14 @@ export interface IUniverseImports {
   fetchCount: (options?: EntityFetchOptions) => Promise<{ count: number }>
 }
 
+export interface IUniverseMessageSubscriptions {
+  fetch: ((options?: UniverseFetchOptions & { raw: true }) => Promise<messageSubscription.MessageSubscriptionRawPayload[] | undefined>)
+  & ((options?: UniverseFetchOptions) => Promise<messageSubscription.MessageSubscription[] | undefined>)
```
[the remark this answers is not in the document]
Comment (artdnlv) [reply src/universe/index.ts:304]: TBH - no idea, how to improve that. I'm open to suggestions.

File: src/entities/message-template/message-template.ts (hunk 1 above)
Comment (artdnlv) [reply src/entities/message-template/message-template.ts:47]: Fixed, thanks for pointing out. Apparently `text` is the only legal value for the type, but `string` looks more future-proof.

File: src/universe/index.ts (hunk 2 above)
[the remark this answers is not in the document]
Comment (datrinh) [reply src/universe/index.ts:304]: I lack context to find a proper naming, but something along the lines of: 
```ts
type fetch1 = ((options?: UniverseFetchOptions & { raw: true }) => Promise<messageSubscription.MessageSubscriptionRawPayload[] | undefined>)
type fetch2 = ((options?: UniverseFetchOptions) => Promise<messageSubscription.MessageSubscription[] | undefined>)

export interface IUniverseMessageSubscriptions {
    fetch: fetch1 & fetch2
}
```

**Pull request merged**